stempy/image.h

```diff
@@ -64,6 +64,11 @@ namespace stempy {
   template <typename InputIt>
   RadialSum<uint64_t> radialSum(InputIt first, InputIt last, int scanWidth = 0, int scanHeight = 0,
         int centerX = -1, int centerY = -1);
-}
+
+  // bins for histogram
+  std::vector<double> getContainer(const STEMImage& inImage, const int numBins);
+  // histogram and bins for the input STEM image
+  std::vector<int> createSTEMHistogram(const STEMImage& inImage, const int numBins, const std::vector<double> bins);
```

Comment:
Let's run a `git-clang-format master` one time before merging to fix the formatting.

Reply:
I believe that I've run `clang-format` on `image.h` before, also I hesitate to do it again because last time it changed lots of formats with the current old code and I had to manually change it back to what it was.

Reply:
Let's just plan on a future run of clang-format on source files if this is the only remaining issue. Here it is clear that your line length is too high, I also suspect we should be preferring const ref for `bins` to avoid a copy.

Reply:
Try using `git-clang-format` instead. It only applies the formatting to your own changes to the code. Make sure your master branch is up-to-date, and then run:

`git-clang-format master`

from inside your branch. You should get changes like the following:
```
diff --git a/stempy/image.cpp b/stempy/image.cpp
index 6e27107..06f92f4 100644
--- a/stempy/image.cpp
+++ b/stempy/image.cpp
@@ -17,13 +17,13 @@ using ArrayHandleView = vtkm::cont::ArrayHandleView<vtkm::cont::ArrayHandle<T>>;
 
 #endif
 
+#include <algorithm>
+#include <cmath>
 #include <future>
 #include <iostream>
 #include <memory>
 #include <numeric>
 #include <sstream>
-#include <cmath>
-#include <algorithm>
 
 using namespace std;
 
@@ -278,7 +278,8 @@ std::vector<double> getContainer(const STEMImage& inImage, const int numBins)
   int height = inImage.height;
   auto curData = inImage.data;
 
-  auto result = std::minmax_element(curData.get(), curData.get() + width * height);
+  auto result =
+    std::minmax_element(curData.get(), curData.get() + width * height);
   double min = *result.first;
   double max = *result.second;
 
@@ -296,7 +297,9 @@ std::vector<double> getContainer(const STEMImage& inImage, const int numBins)
 
 // function that computes histogram for all the STEM images
 // each histogram is a vector<int>
-std::vector<int> createSTEMHistogram(const STEMImage& inImage, const int numBins, const std::vector<double> bins)
+std::vector<int> createSTEMHistogram(const STEMImage& inImage,
+                                     const int numBins,
+                                     const std::vector<double> bins)
 {
   // initialize output
   std::vector<int> frequencies(numBins, 0);
@@ -317,7 +320,7 @@ std::vector<int> createSTEMHistogram(const STEMImage& inImage, const int numBins
     }
     // the max value is put in the last slot
     if (value == bins[numBins]) {
-        ++frequencies[numBins-1];
+      ++frequencies[numBins - 1];
     }
   }
 
diff --git a/stempy/image.h b/stempy/image.h
index 6ac184e..854a0eb 100644
--- a/stempy/image.h
+++ b/stempy/image.h
@@ -68,7 +68,9 @@ namespace stempy {
   // bins for histogram
   std::vector<double> getContainer(const STEMImage& inImage, const int numBins);
   // histogram and bins for the input STEM image
-  std::vector<int> createSTEMHistogram(const STEMImage& inImage, const int numBins, const std::vector<double> bins);
-} // namespace stempy
+  std::vector<int> createSTEMHistogram(const STEMImage& inImage,
+                                       const int numBins,
+                                       const std::vector<double> bins);
+  } // namespace stempy
 
 #endif
```